drivers/net/ethernet/intel/ice/ice_flow.c

```diff
@@ -2491,8 +2775,10 @@ ice_rem_rss_cfg_sync(struct ice_hw *hw, u16 vsi_handle,
 	int status;
 
 	segs_cnt = (cfg->hdr_type == ICE_RSS_OUTER_HEADERS) ?
-			ICE_FLOW_SEG_SINGLE : ICE_FLOW_SEG_MAX;
-	segs = kcalloc(segs_cnt, sizeof(*segs), GFP_KERNEL);
+			   ICE_FLOW_SEG_SINGLE :
+			   ICE_FLOW_SEG_MAX;
+	segs = devm_kcalloc(ice_hw_to_dev(hw), segs_cnt, sizeof(*segs),
+			    GFP_KERNEL);
```

Comment:
please undo all unintentional kmalloc() family calls replaced by devm_kmalloc()
OOT uses devm_ too much in the SHARED/ directory